src/common/components/form/multiDropdownField.module.scss

```diff
@@ -0,0 +1,12 @@
+.hideLabel {
+  label {
+    position: absolute;
+    width: 1px;
+    height: 1px;
+    padding: 0;
+    margin: -1px;
+    overflow: hidden;
+    clip: rect(0, 0, 0, 0);
+    border: 0;
+  }
+}
```

Comment:
You could have mixin for the label if you need sr label in multiple components e.g. https://github.com/City-of-Helsinki/linkedcomponents-ui/blob/develop/src/assets/styles/mixins.scss